sdk/python/endpoints/batch/heart-classifier-mlflow/code/batch_driver_parquet.py

```diff
@@ -0,0 +1,30 @@
+import os
+import mlflow
+import pandas as pd
+from pathlib import Path
+
+
+def init():
+    global model
+    global output_path
+
+    # AZUREML_MODEL_DIR is an environment variable created during deployment
+    # It is the path to the model folder
+    # Please provide your model's folder name if there's one:
+    model_path = os.path.join(os.environ["AZUREML_MODEL_DIR"], "model")
+    output_path = os.environ["AZUREML_BI_OUTPUT_PATH"]
```

Comment:
what's AZUREML_BI_OUTPUT_PATH?

Reply:
It's the path where batch deployments read data to. If you want to write batch score scripts that write files, then you can write directly. This has been undocumented since ever.